source/dyaml/scanner.d

```diff
@@ -740,8 +745,8 @@ struct Scanner
         {
             // Check one char first, then all 3, to prevent reading outside the buffer.
             return reader_.column     == 0     &&
-                   reader_.peekByte() == '.'   &&
-                   reader_.prefix(3)  == "..." &&
+                   reader_.front == '.'   &&
+                   reader_.slice(3)  == "..." &&
```

Comment:
how about supporting the true slicer operator, e.g: `reader_[0..3]` ?

Reply:
true slices don't actually cover the same functionality. `.slice(x)` would translate to something more like reader_[curPos..curPos+x], but with decoding. My goal is to have it removed.

Reply:
okay, dismiss the comment then